deployment/lib/lambdas/cf-url-rewriter.ts

```diff
@@ -0,0 +1,70 @@
+import { CloudFrontRequestEvent, CloudFrontRequest } from 'aws-lambda';
+
+const https = require('https');
+
+export async function handler(event: CloudFrontRequestEvent, context, callback): Promise<CloudFrontRequest> {
+  const request = event.Records[0].cf.request;
+
+  console.log('request', JSON.stringify(request));
+
+  if (request.uri.includes("latest")) {
+
+    const indexUri = request.uri.replace(/latest.*/, 'dist/index.json');
```

Comment:
This will match `foolatestbar`, include `/` in the regex.

Reply:
Sure